runtime/onert/core/src/util/shapeinf/Reshape.cc

```diff
@@ -67,15 +85,27 @@ void StaticInferer::visit(const ir::operation::Reshape &op)
   const auto shape_idx{op.getInputs().at(ir::operation::Reshape::Input::SHAPE)};
   const auto &shape = _operands.at(shape_idx);
 
-  // if shape is from Const, TFLC put the shape of output into tensor
+  ir::Shape new_shape;
   if (shape.isConstant())
   {
-    // no change on output shape
-    return;
```

Comment:
The reason I wrote code here is that in this case TFLCovnerter writes const values into the output tensor.
So, I wonder the change is necessary.

Reply:
There is no special reason. I would like to use returning as little as possible.
If you want revert it, I will do that.

Reply:
hmm.. let's keep the changed code. who knows tflite converter changes their behavior!